### ISSUE TITLE
New blue board support ?

### ISSUE THREAD

**Issue by nicolas-van** [silver13/H8mini_blue_board#22]
Hello,

Is there any plan to support the new blue board ? Or do you know someone that managed to flash it ?

Thanks in advance for your answer ;-)

**Comment by ImDroning** [comment]
Yes I’m having the same troubles. I have managed to read the new code using stm32box software. I can not erase or download .bin file. The device shows up as a stm32L4Qxx or stm32L4Pxx . STM32L is very different microchip. Keil software says not st product.

**Comment by ImDroning** [comment]
This is the link where I realized I was in over my head 
https://stackoverflow.com/questions/43640408/c-code-modification-from-stm32f-to-stm32l

**Comment by ImDroning** [comment]
The BIN file I pulled out with STM32_CubeProgrammer (my mistake calling it STM32Box) github doesn't support this type of file being copied.  when I convert it to text file it looks like this
  ]  a  Õ                               e          g  i  ‡  k  k  k  k  k  Ç  k  -  E  a  k  k  ?  Y  Ã  k  k  Y  ­  k  k  k  k  k  k  k  k  k  k  k  k  H…Fðàû H G#     qFBÐïó	€àïó€qF JGÏ&  pGµJjÁHÕ ! àFL!`i‰‰	Ð)Ð)Ð)ÐàH àH‘iŠ’!‘@ðú	I``J€ð‡ú `½    P lÜ    '   $ô @B H Gþçþçþçþçþçþçq  pµJ I@LÓC
`h–BÑ`hžBÐ`à ¨BòÓH`ÿ÷ÿp½ü  U3U3¥¥¥¥""pµY!L ˆ"!` `"`#h +ùÐHÀ8AhJ@A`Ah"CA`Hh‰
‰`!a‰`H %E`H@0E`H€0E`
HÀ0E`HE`I@1H`0 ‰ˆcÿ÷jÿ%`p½ÿ÷Çÿ ðƒøþç  Püüÿü  P @ P A P  pG¤H¥IÀkˆBÑ pG  pGµ¡Lá!I FðCû hð!ˆC 0 `I Fð6ûœIhÿ"C` ™I@`b¶½”IŠiÒ üÕ`pGµF(Ð ð
ù ‘I€ˆaÿ÷ ÿI  `ÿ÷Íÿ½øµ & ðýø‹HÀ8AhŠJ@A`Ah"CA`  Akÿ"2‘CAcAkAcI  `‚OøjD €H€8 jCÿ -0 ðÐú|Od%€?àzHÀjD 8jC´BÐ&Fd%  ðÀúmñÒ Fø½øµsL %¥` ÿ!@ ðœúÿ÷ÿ&(Ñ h0C ` i0C aÿ÷±ÿ(Ðÿ÷­ÿ%-(Ð ðûF  )-Ð/àð1ÿ_L€<¥`  ð‘þ  ð|þ¦` ðÑúÿ÷^ÿ!Jà ðíúÿ÷XÿWI(F ðfú`h (üÐ !  ðcúðTÿ (Ñ h!ÉC `!	/à¡hC¡`!iC!aÿ÷9ÿHH`HI(F ðEú`h (üÐFH@hÿ÷Nÿ%

Again Way over my head.
Please take the H8 mini off the NFE silverware upgade list if it is not compatible with the coding

Thank you for all your help, Imdroning

**Comment by Zapeth** [comment]
I messed around with this as well some time ago, without much results. But today when I wanted to check if there have been any breakthroughs since then, I stumbled upon the datasheet of the chip -> http://bbs.panchip.com/forum.php?mod=viewthread&tid=36 (I don't think this link was available back then, there also seems to be a flashing description?)

And while I'm here I guess I might as well write about my other findings from back then:
I used an ST-Link V2 for connecting to the board, though couldn't really get far with the recommended GUI tools.

After messing/searching around a bit (I was and still am fairly unexperienced with this kind of hardware interfacing) I learned about openocd and was able to figure out the idcode of the board being `0x0bb11477`, indicating an stm32f0x-like configuration.

Using openocd's `numicro` target (its not really this target, but the configuration is similar enough) I was also able to connect to the board and issue commands, like halting/resuming the cpu and reading registers/memory.

The parts that I was able to dump weren't really useful back then, but now with the datasheet they might be, so I'll attach the most interesting parts here -> [dumps.zip](https://github.com/silver13/H8mini_blue_board/files/5211432/dumps.zip) (openocd might have automatically swapped the data, so I'm not sure if the endianness is correct)

Unfortunately I was unable to locate the `SCS_CPUID` value anywhere in the dumped memory, the value at the documented memory offset reads `0x410cc200` (should be `0x0bb11477`), though maybe I'm not reading it correctly.

Anyway, hope this helps in some way. It would be nice if this board could still be hacked with custom firmware for some acro flying :)

PS: @ImDroning Text encoding is not really useful for binary data, you might want to put that file in a zip archive and upload that instead (or just rename the extension to something Github allows)
